<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="ui" native="true">
       <widget class="QWidget" name="HeartWave" native="true">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>60</y>
          <width>311</width>
          <height>441</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget 
{
border:2px solid rgb(0, 0, 0);
background-color: rgb(200, 200, 200);
}
</string>
        </property>
        <widget class="QPushButton" name="upButton">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>240</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
image: url(:/buttons/upButton.svg);
border-radius: 15px;
} 
QPushButton:pressed {
	background-color: #ffd300;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="okButton">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>280</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
image: url(:/buttons/okButton.svg);
border-radius: 15px;
}
QPushButton:pressed {
	background-color: #ffd300;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="rightButton">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>280</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
image: url(:/buttons/rightButton.svg);
border-radius: 15px;
}
QPushButton:pressed {
	background-color: #ffd300;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="downButton">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>320</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
image: url(:/buttons/downButton.svg);
border-radius: 15px;
}
QPushButton:pressed {
	background-color: #ffd300;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="leftButton">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>280</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
image: url(:/buttons/leftButton.svg);
border-radius: 15px;
}
QPushButton:pressed {
	background-color: #ffd300;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="backButton">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>230</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
image: url(&quot;:/buttons/backButton.svg&quot;);
border-radius: 15px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="menuButton">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>230</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
image: url(:/buttons/menuButton.svg);
border-radius: 15px;
}
QPushButton:pressed {
	background-color: #ffd300;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="powerButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>380</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
image: url(:/buttons/powerButton.svg);
border-radius: 15px;
}
QPushButton:pressed {
	background-color: #ffd300;
	
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="coherenceLevel">
         <property name="geometry">
          <rect>
           <x>83</x>
           <y>20</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QWidget" name="offView" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>60</y>
           <width>231</width>
           <height>161</height>
          </rect>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">border:2px solid rgb(0, 0, 0);
background-color: rgb(0, 0, 0);</string>
         </property>
         <widget class="QWidget" name="screenView" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>231</width>
            <height>161</height>
           </rect>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px solid rgb(0, 0, 0);
background-color: rgb(114, 207, 159);</string>
          </property>
          <widget class="QProgressBar" name="batteryLevel">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>5</y>
             <width>50</width>
             <height>10</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>7</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">selection-background-color: rgb(164, 0, 0);
background-color: rgb(0, 0, 0);
color:rgb(255,255,255);
border:0px</string>
           </property>
           <property name="value">
            <number>75</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="invertedAppearance">
            <bool>true</bool>
           </property>
           <property name="textDirection">
            <enum>QProgressBar::TopToBottom</enum>
           </property>
          </widget>
          <widget class="QLabel" name="hrContact">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>212</x>
             <y>2</y>
             <width>16</width>
             <height>16</height>
            </rect>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">border:0px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../res/HeartWave.qrc">:/buttons/hrContact.svg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QListWidget" name="menuListView">
           <property name="geometry">
            <rect>
             <x>2</x>
             <y>40</y>
             <width>227</width>
             <height>119</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border:0px;
background-color: rgb(255,255,255);</string>
           </property>
           <property name="layoutMode">
            <enum>QListView::SinglePass</enum>
           </property>
          </widget>
          <widget class="QLabel" name="menuLabel">
           <property name="geometry">
            <rect>
             <x>2</x>
             <y>20</y>
             <width>227</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border:0px;
background-color: rgb(114, 159, 207);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>MAIN MENU</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="bpView">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>80</y>
             <width>131</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border:0px;
background-color: rgb(255,255,255);
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QWidget" name="sessionView" native="true">
           <property name="geometry">
            <rect>
             <x>2</x>
             <y>40</y>
             <width>227</width>
             <height>119</height>
            </rect>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">border:0px;
background-color: rgb(255,255,255);</string>
           </property>
           <widget class="QCustomPlot" name="graphView" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>20</y>
              <width>227</width>
              <height>81</height>
             </rect>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">border:0px;
background-color: rgb(255,255,255);</string>
            </property>
           </widget>
           <widget class="QLabel" name="coherenceLabel">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Co</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="timeLabel">
            <property name="geometry">
             <rect>
              <x>75</x>
              <y>0</y>
              <width>21</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Ti:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="achievementLabel">
            <property name="geometry">
             <rect>
              <x>153</x>
              <y>0</y>
              <width>21</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Ach:</string>
            </property>
           </widget>
           <widget class="QLabel" name="coherenceVal">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>0</y>
              <width>41</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="timeVal">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>0</y>
              <width>61</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>00:00</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="achievementVal">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>0</y>
              <width>51</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QProgressBar" name="coherenceBar">
            <property name="geometry">
             <rect>
              <x>-3</x>
              <y>102</y>
              <width>231</width>
              <height>16</height>
             </rect>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QLabel" name="lowCoherence">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>100</y>
              <width>71</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="medCoherence">
            <property name="geometry">
             <rect>
              <x>73</x>
              <y>100</y>
              <width>71</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="highCoherence">
            <property name="geometry">
             <rect>
              <x>153</x>
              <y>100</y>
              <width>71</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </widget>
          <zorder>menuListView</zorder>
          <zorder>batteryLevel</zorder>
          <zorder>menuLabel</zorder>
          <zorder>hrContact</zorder>
          <zorder>sessionView</zorder>
          <zorder>bpView</zorder>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="adminTab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Admin</string>
       </property>
       <widget class="QLabel" name="power">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>36</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Power</string>
        </property>
       </widget>
       <widget class="QLabel" name="breathPacer">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>62</y>
          <width>73</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Breath Pacer</string>
        </property>
       </widget>
       <widget class="QComboBox" name="hrContactVal">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>94</y>
          <width>111</width>
          <height>25</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>FALSE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TRUE</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="chargeButton">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>30</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
image: url(:/buttons/chargeButton.svg);
border-radius: 15px;
}

QPushButton:pressed {
	background-color: green;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="heartContact">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>94</y>
          <width>64</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>HR Contact</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="batteryVal">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>30</y>
          <width>111</width>
          <height>26</height>
         </rect>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
       <widget class="QSpinBox" name="breathPacerVal">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>62</y>
          <width>111</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="testDataSelectVal">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>530</y>
          <width>131</width>
          <height>25</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Low Coherence</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High Coherence</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="testDataSelect">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>530</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Test Data Selection</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>src/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/HeartWave.qrc"/>
 </resources>
 <connections/>
</ui>
